meeting-minutes/2024-10-16.md

```diff
@@ -0,0 +1,147 @@
+
+### 1.7.2 Members attaining voting rights at the end of this meeting
+
+### 1.7.3 Members losing voting rights if they have not joined this meeting by the time it ends
+
+### 1.7.4 Members who previously lost voting rights who are attending this meeting
+
+### 1.7.5 Members who have declared a leave of absence
+
+# 2. Future Meetings
+
+## 2.1 Future meeting schedule (Secretary)
+
+- Scheduled Teleconferences (Wednesday at 09:00 PDT / 12:00 EDT / 18:00 CEST / **16:00** UTC for 1 hour)
+
+    ```
+    November 29, 2024 (normal time switch back done, same time on wall clock)
+    December 18, 2024
+    ```
+
+# 3. Discussion
+
+- [Using "End" word in the lifecycle definitions #38](https://github.com/oasis-tcs/openeox/issues/38):
+  - Przemyslaw presents the rationale behind the ticket
+  - Kris is also in favor of using the concept of milestones eg. a start of a window is a milestone and the end is another milestone
+  - Feng, Omar, and Stefan like to focus on end "markers" only
+  - Feng and Omar mention scope problems with theb charter
+  - Stefan Hagen sees enough work to get the end "markers" right as some may depend on start of contract
+  - Thomas Schmidt is open for adding the start "marker" but is sceptical if this is doable and useful - maybe add when need clearly arises
+  - all continue to discuss if adding a start "marker" is useful already and if it fits the scope given by the charter
+  - Thomas Schmidt identified another discussion oging on that is about successors taking over
+  - Thomas Schmidt also sees the need to agree on statements on precision to accomodate for risky product deployments where there might
+    be a possible future need to shorten a support range 
+  - Omar encourages all participants to add comments to the issue and suggests for now to focus on the end of stages and maybe later come back
+
+- [Suggested new optional field "successor" #3](https://github.com/oasis-tcs/openeox/issues/3):
+  - Thomas Schmidt shortly presents the issue
+  - Stefan Arntzen and Stefan Hagen are not against an optional field but have doubts about the use
+  - Stefan Hagen esp. does not expect "hard" benefit from values in that field
+  - Omar is in favor as an optional field even useful to some when most may not use it
+  - Shridar esp. sees that in the end of "x" announcement from a supplier there is a place to state successors
+  - Kris suggests to rename as "recommended action" (or similar) instead
+  - all continue to discuss
+  - Shridar moves to consider an optional field in the targeted version of the specification
+  - Omar objects the idea to have the name of the field unspecified
+  - Stefan Hagen suggests to add more complex information (like recommended action) later to another field
+  - Thomas Schmidt moves to add an optional field successor into the targeted version of the specification
+    - Stefan Hagen seconds
+    - discussion:
+      - Jautau does not think the field should be optional 
```

Comment:
```suggestion
      - Jautau does not think the field should be optional and wants to see it as mandatory field

Reply:
I prefer leaving the statement as minuted because it includes already the proposed addition and Jay might have a say in that too.